Makefile

```diff
@@ -0,0 +1,37 @@
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+build-base-sys-dl: ## build orbyter-base-sys-dl
+	docker build -t orbyter-base-sys-dl orbyter-base-sys-dl
+
+release-base-sys-dl: ## tags git with orbyter-base-sys-dl-<VERSION> and pushes to github.
+	git tag orbyter-base-sys-dl-`cat orbyter-base-sys-dl/VERSION`
```

Comment:
What if I forget to increment the VERSION file? Will this fail?

Reply:
For now, bumping the version is manual. I'd have to play around with something like [bumpversion](https://github.com/peritus/bumpversion) to automate this. It's a little tricky because there are several versions in this repo.